interface.js

```diff
@@ -0,0 +1,61 @@
+let news = new News();
+const altImage = "/Users/kikidawson/Desktop/Projects/week-7/news-summary-challenge/images/blank.png"
+
+function getNewsData() {
+  return fetch("http://news-summary-api.herokuapp.com/guardian?apiRequestUrl=http://content.guardianapis.com/search?show-fields=all").then(response => {
+    return response.json();
+  })
+}
+
+getNewsData().then(news => {
+  let newsData = news.response.results
+  saveAll(newsData)
+});
+
+function saveAll(articles) {
```

Comment:
really nice that this is split into separate functions, getting data first, then doing something with it.